### ISSUE TITLE
p8b updates for workflow 

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/global-workflow#641]
**Description**
Update to configuration and possible scripts for additional fix files to allow for the running of p8b free forecasts atm only and for atm-ocn-ice-wav (aerosols is separate issue).  This will include a model update which has not been updated yet. 
 
**Requirements**

**Acceptance Criteria (Definition of Done)**
Be able to run free-forecast atm only and atm-ocn-ice-wave with p8b settings. 

**Dependencies**
https://github.com/ufs-community/ufs-weather-model/issues/1042 will need to be merged first.

**Comment by JessicaMeixner-NOAA** [comment]
A tag of ufs-weather-model has been created: https://github.com/ufs-community/ufs-weather-model/tree/Release/P8b and the GOCART PR has been merged.  I'm merging in the changes to my branch: https://github.com/jessicameixner-noaa/global-workflow/tree/feature/p8b and hope to have a PR submitted ASAP.
